src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -85,7 +86,7 @@ const Payment = getPaymentModel(mongoose)
 // NOTE: Refer to this for documentation: https://github.com/sideway/joi-date/blob/master/API.md
 const Joi = BaseJoi.extend(JoiDate)
 
-const submitEncryptModeForm = async (
+export const submitEncryptModeForm = async (
```

Comment:
Would avoid exposing this function as an export. `export`-ed functions means that the function is free to be called by others, however, in this PR we're not trying to allow this, but to expose it for our test scripts to execute it.

Currently in our repo, we are doing it by creating a new constant named [<function_name>ForTest](https://github.com/search?q=repo%3Aopengovsg%2FFormSG+fortest&type=code) -- there's some inconsistency there. Can open a separate PR to make them consistent (would prefer `xxxForTest`).